lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1646,82 +1630,68 @@ LowererMD::LowerExitInstr(IR::ExitInstr * exitInstr)
             exitInstr);
     }
 
+    int32 regPopedSize = 0;
+
     // 2. Restore saved double registers. Generate vpop {d8-d15}
     BVUnit savedDoubleRegs(this->m_func->m_unwindInfo.GetDoubleSavedRegList());
     if (!savedDoubleRegs.IsEmpty())
     {
-        IR::Instr * instrVPop = IR::Instr::New(Js::OpCode::VPOP, this->m_func);
-        instrVPop->SetDst(IR::RegBVOpnd::New(savedDoubleRegs, TyMachReg, this->m_func));
-        instrVPop->SetSrc1(IR::IndirOpnd::New(IR::RegOpnd::New(nullptr, RegSP,TyMachReg, this->m_func), (int32)0, TyMachReg, this->m_func));
-        exitInstr->InsertBefore(instrVPop);
+        for (RegNum reg = FIRST_CALLEE_SAVED_DBL_REG; reg <= LAST_CALLEE_SAVED_DBL_REG; reg = (RegNum)(reg + 1))
```

Comment:
Same comment as above, use LDP to load in pairs for efficiency/code size. It can be a TODO for now, but should be notated.

Reply:
Also since this is the floating point case, use FLDR instead of LDR.

Reply:
Added ToDo comment

---
In reply to: [143539271](https://github.com/Microsoft/ChakraCore/pull/3903#discussion_r143539271) [](ancestors = 143539271)